src/sidebar/media-embedder.js

```diff
@@ -25,11 +25,47 @@ function iframe(src) {
   return iframe_;
 }
 
+/**
+ * Return a formatted YouTube URL query string containing allowed params
+ * e.g. '?t=5'
+ */
+function youTubeQueryParams(link) {
+  var query;
+  // Whitelisted URL query parameters
+  // See https://developers.google.com/youtube/player_parameters
+  const allowedParams = [
+    'end',
+    'start',
+    't',                // Alias for `start`
+  ];
+  const linkParams = queryString.parse(link.search);
+  const filteredQuery = {};
+  // Filter linkParams for allowed keys and build those entries
+  // into the filteredQuery object
+  Object.keys(linkParams)
+    .filter(key => allowedParams.includes(key))
+    .forEach(key => {
+      if (key === 't') {
+        // `t` is a unique param in the YouTube URL API
+        // It works in URLs but not embeds; those require the `start` param
+        // instead. But `t` is commonly used in URLs, so it's worthwhile to
```

Comment:
It appears that "t" is not just an alias for start but it also accepts different formats. eg. YouTube accepts "t=1m30s" or "t=90s" or "t=90", whereas it looks like `start` accepts just a number in seconds.

We could punt on this for an initial implementation, but since the "Share" button on YouTube videos generates links of the first form (eg. https://youtu.be/{id}?t=1m30s) people are likely to run into it. Thoughts?

Reply:
Well, that was a neat little trick. Thanks for noticing this! I've implemented support for it.